mlrun/execution.py

```diff
@@ -1041,7 +1041,11 @@ def set_if_not_none(_struct, key, val):
 
         # completion of runs is not decided by the execution as there may be
         # multiple executions for a single run (e.g. mpi)
-        if self._state != "completed":
+        run_kind = self.labels.get(mlrun_constants.MLRunInternalLabels.kind, "")
```

Comment:
update the comment above it, explaining the reasoning to allow changing run state for local